broker/l2tp_broker.py

```diff
@@ -792,7 +792,15 @@ def hook(self, name, *args):
     # Execute the registered hook
     logger.debug("Executing hook '%s' via script '%s %s'." % (name, script, str([str(x) for x in args])))
     try:
-      gevent_subprocess.call([script] + [str(x) for x in args])
+      script_process = gevent.subprocess.Popen([script] + [str(x) for x in args], stdout=gevent.subprocess.PIPE, stderr=gevent.subprocess.PIPE)
+      stdout, stderr = script_process.communicate()
+
+      if stdout:
+          logger.debug("Hook '%s' script stdout:" % script)
```

Comment:
This code file is still using 2 spaces to indent, so you should follow this style. I agree that we should migrate to 4 spaces in the future and make the whole thing PEP8 compliant. But at the moment, please make it the same style as other code.